collect_app/src/main/res/drawable/form_state_submission_failed.xml

```diff
@@ -0,0 +1,17 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="40dp"
+        android:height="40dp"
+        android:viewportWidth="120"
+        android:viewportHeight="120">
+
+    <path
+            android:pathData="M60,0C26.88,0 0,26.88 0,60c0,33.12 26.88,60 60,60 33.12,0 60,-26.88 60,-60 0,-33.12 -26.88,-60 -60,-60"
+            android:fillColor="#F44336"/>
```

Comment:
Please use `@color/red` here instead of `#F44336` since exactly the same color is already defined in our colors.xml file.

Reply:
👍 I'll make all these new colors into variables like this one for consistency.